### DISCUSSION TITLE
SortIdents not in v1.14.0?

### DISCUSSION THREAD

**Discussion by sagrimm0** [stuart-lab/signac#1816] [Q&A]
I updated to Signac v1.14.0 so I could try out the new SortIdents function (which the Changelog says was added), but I'm getting errors saying the function isn't found.  Was it assigned a different name or is it really not there?

**Comment by timoast** [comment]
https://github.com/stuart-lab/signac/blob/0b33b1154f9a610897d1efad2c0065081c4e7132/NAMESPACE#L177

You should double check your installed package version
